packages/terra-form-select/src/combobox/Frame.jsx

```diff
@@ -208,6 +208,7 @@ class Frame extends React.Component {
       className: cx('search-input'),
```

Comment:
> Doing this would also allow you to remove all the aria-labels scattered throughout.

There's a note on the issue that mentions this change would allow us to remove the aria-labels. This will need to be tested with a screen reader to ensure it still works as expected, but we may want to remove those labels with this change.

Reply:
Earlier `aria-label` was read twice by the screen because it was rendering for both `select div` and the `input field`. I have already removed the aria-label for the select div. Removing the label for input field gives axe violations.I don't think that is what we want.
@StephenEsser

Reply:
We do not want any axe violations if the aria-label is required on the input, I see that you were able to remove the aria-labels from the div and perhaps that is the only one we'll be able to remove